JuiceMaker/JuiceMaker/Controller/ManageStockViewController.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import UIKit
+
+class ManageStockViewController:
+    
+    UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    @IBAction func closeManageStockView(_ sender: Any) {
```

Comment:
Button의 액션으로만 해당 메서드를 탈 수 있도록 Any 타입 보다는 UIButton 타입으로 변경해주는걸 추천 드려요 ㅎㅎ

```suggestion
    @IBAction func closeManageStockView(_ sender: UIButton) {
```

Reply:
UIButton 타입으로 변경하였습니다.